subprojects/language-native/src/main/resources/META-INF/gradle-plugins/org.gradle.swift-executable.properties

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 #
 
-implementation-class=org.gradle.language.swift.plugins.SwiftExecutablePlugin
+implementation-class=org.gradle.language.swift.plugins.SwiftApplicationPlugin
```

Comment:
this file should be deleted.

Reply:
Good point, I thought the same workaround as C++ was required.

Reply:
Now GitHub thinks the file was named, but the original file was removed.